deploy/helm/kubecf/assets/scripts/jobs/postgres/bpm_process.sh

```diff
@@ -12,6 +12,8 @@ wait_for_file() {
 # shellcheck disable=SC1091
 source /var/vcap/jobs/postgres/bin/pgconfig.sh
 
+chmod -R 700 /var/vcap/store/postgres/
```

Comment:
Always good to be explicit. :)

```suggestion
chmod --recursive 0700 /var/vcap/store/postgres/
```